contributors/devel/sig-release/cherry-picks.md

```diff
@@ -27,7 +27,7 @@ branches.
 - Have `hub` installed, which is most easily installed via
   `go get github.com/github/hub` assuming you have a standard golang
   development environment.
-- A github token which has permissions to create a PR in an upstream branch.
+- A github personal access token which has permissions to access public repositories.
```

Comment:
also permission to create a brunch?

Reply:
Me too, this is the token I recently used:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/1494193/114781172-313b6f80-9d2d-11eb-9f55-1b672852975f.png">

I don't think you need all of that, lemme do a quick test with just "public_repo" enabled to confirm...

Reply:
Confirmed that this is all you need:

<img width="420" alt="image" src="https://user-images.githubusercontent.com/1494193/114781977-4369dd80-9d2e-11eb-8383-fa0c0359bd85.png">

A PAT w/ the above perms was used to create this a few mins ago:

https://github.com/kubernetes/kubernetes/pull/101128